use-timescale/popsql.md

```diff
@@ -0,0 +1,154 @@
+---
+title: PopSQL
+excerpt: Use PopSQL to run SQL queries, create charts and dashboards, and collaborate with teammates.
+keywords: [popsql, sql editor, chart, dashboard]
+---
+
+# PopSQL
+
+You use Timescale's [PopSQL](https://popsql.com) editor to write queries, visualize data, and share your results. 
+
+![Screenshot of PopSQL](https://assets.timescale.com/docs/images/popsql_product_screenshot.png)
+
+Available features are:
+
+- **Cross platform**:  Works as a [web app](https://app.popsql.com) or for [desktop](https://popsql.com/download) on macOS, Windows, and Linux.
+- **Easy connection**: To PostgreSQL, Timescale, Redshift, Snowflake, BigQuery, MySQL, SQL
+  Server, [and more](https://popsql.com/connections)
+- **Real-time collaboration**: Work with your team directly in the SQL query
+  editor with live presence and multiple cursors.
+- **[Schema browser](https://docs.popsql.com/docs/schema)**: Understand the
+  structure of your database and see usage data on tables and columns.
+- **Autocomplete**: Get suggestions as you type your queries.
+- **[Version history](https://docs.popsql.com/docs/version-history)**: Use the
```

Comment:
I would clarify version history is for queries. I didn't understand what this was for when I read it.

Reply:
Done